### PR TITLE
Max assertion age configurable

### PR DESCRIPTION
**Pull request by tmulkern**
- I added the code necessary to allow the Max Assertion Age to be changed.
- Updated spring-security-saml2-core dependency from 1.0.0.RELEASE to 1.0.2.RELEASE

### CODE REVIEW

**Comment by tmulkern** [comment]
This pull request is needed to support changes to the jira-saml-plugin [Pull Request](https://github.com/bitium/jira-saml-plugin/pull/40) and to the confuence-saml-plugin [Pull Request](https://github.com/bitium/confluence-saml-plugin/pull/55)

**Comment by tmulkern** [comment]
@rkarodia bump

File: src/main/java/com/bitium/saml/config/SAMLConfig.java [hunk 1]
```diff
@@ -103,7 +112,7 @@ public String getIdpEntityId() {
 
     public String getUidAttribute() {
         return StringUtils.defaultString((String)pluginSettings.get(UID_ATTRIBUTE_SETTING), "NameID");
-    }
+    }l
```
Comment (rkarodia) [inline src/main/java/com/bitium/saml/config/SAMLConfig.java]: Apologies for the delay. There's a typo here.

Comment (tmulkern) [inline src/main/java/com/bitium/saml/config/SAMLConfig.java]: Sorry @rkarodia I must be blind, where is the typo?

Comment (rkarodia) [inline src/main/java/com/bitium/saml/config/SAMLConfig.java]: The "l" on line 115

Comment (tmulkern) [inline src/main/java/com/bitium/saml/config/SAMLConfig.java]: @rkarodia I don't see the typo in my code
https://github.com/tmulkern/atlassian-saml/blob/max-assertion-age-configurable/src/main/java/com/bitium/saml/config/SAMLConfig.java
Where is this one coming from??

Comment (tmulkern) [inline src/main/java/com/bitium/saml/config/SAMLConfig.java]: @rkarodia  Ok, I saved the file with an extra line and that seemed to get rid of it, very strange

Comment (rkarodia) [inline src/main/java/com/bitium/saml/config/SAMLConfig.java]: @tmulkern your pull request doesn't include the commit where you fixed it https://github.com/tmulkern/atlassian-saml/commit/8af68bc9ac3c2474f3c345e96eaaa71720cb68c8#diff-30c554871a46128405ed990df6369d1a

Comment (tmulkern) [inline src/main/java/com/bitium/saml/config/SAMLConfig.java]: @rkarodia Yes it does https://github.com/bitium/atlassian-saml/pull/5/commits/8af68bc9ac3c2474f3c345e96eaaa71720cb68c8

Comment (tmulkern) [inline src/main/java/com/bitium/saml/config/SAMLConfig.java]: All changes in max-assertion-age-configurable branch on tmulkern/atlassian-saml _should_ be included in this pull request

**Comment by papageod** [comment]
Any guess on when we might see this implemented? The current hard-coded 7200 second token life is a major problem for our team.

**Comment by tmulkern** [comment]
@papaged I am just waiting for @rkarodia to approve this

**Comment by tmulkern** [comment]
@rkarodia thanks

**Comment by tmulkern** [comment]
@papageod the code has been merged

**Pull request merged**